firestore/angular-rewrite/package.json

```diff
@@ -4,6 +4,8 @@
   "scripts": {
     "ng": "ng",
     "start": "(cd functions && npm run build) && firebase --project demo-friendly-eats emulators:exec --import ./imported-firebase-data 'ng serve'",
+    "emulators": "(cd functions && npm run build) && firebase --project demo-friendly-eats emulators:exec --import ./imported-firebase-data 'ng serve -c local'",
```

Comment:
let's move this build step out as a pre-build step and call it from the 3 scripts